cfme/tests/login/test_ssui_login.py

```diff
@@ -0,0 +1,17 @@
+import pytest
+
+from cfme.base.ssui import SSUIBaseLoggedInPage
+from cfme.utils.appliance import ViaSSUI
+from cfme.utils.version import current_version
+
+
+@pytest.mark.uncollectif(lambda: current_version() < '5.8')
+@pytest.mark.parametrize('context', [ViaSSUI])
+def test_ssui_login(appliance, context):
+    """SSUI Login test"""
+
+    with appliance.context.use(context):
+        appliance.server.login()
+        logged_in_page = appliance.browser.create_view(SSUIBaseLoggedInPage)
```

Comment:
at first glance i wonder about using `appliance.browser` here i was under the impression ssui should use a different one in some way

@psav ?

Reply:
Agreed, I believe it's called ```appliance.ssui``` but in all honesty, I forget what the ```context``` is......but we should be instantiating the view from the context, IF the test is to be parametrized on the context. This test, as it is just testing ssui login, doesn't need to use context as a parameter. If only makes sense if we are trying to do both UI and SSUI.